CMakeLists.txt

```diff
@@ -285,22 +285,12 @@ if (SINGULARITY_USE_HDF5)
   enable_language(C)
```

Comment:
@mauneyc-LANL We can drop this for cmake>=3.26, see: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/8015

Reply:
yes; Indeed, we don't require to do this for `spiner`, and I'm working on a PR for singularity to follow that approach.